modules/PhyVersoBSP/phyverso_cli/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.10)
+
+# set the project name
+project(phyverso_cli VERSION 0.1)
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
this could be replaced by 
`target_compile_features(phyverso_cli PRIVATE cxx_std_17)`
after the target has been defined further down

Reply:
both are used through out the repe

Reply:
There's an open PR https://github.com/EVerest/everest-core/pull/651. A comment mentions
> concerning "modern" cmake best practices, the target_** commands should be used instead of setting global flags, because they are more specific and you have more fine grained control.

The PR's aim is to drop the per-module definitions completely, BTW.